basic/stam/StamExAnswer.py

```diff
@@ -0,0 +1,49 @@
+"""This program reads a file, edits it and saves it in a new location"""
+import os
+FIRST_NAME = "Keren"
+LAST_NAME = "Guez"
+
+
+def add_vals(line):
+    """This function concatenates a line with some given text
```

Comment:
functions doc should start with Imperative verb.
For example: Concatenate a line with...
Add something with
etc.